scalalib/test/resources/hello-java/app/testJunit5/src/Junit5Tests.java

```diff
@@ -0,0 +1,20 @@
+package hello;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Disabled;
+
+public class Junit5Tests {
+
+    @Test
```

Comment:
I guess it is because they now support a lot more annotations, inner class tests, parametrized tests etc. https://junit.org/junit5/docs/current/user-guide/#launcher-api-discovery
And if they add anything new, we can just update JUnit version and that's all. (My assumption)

Reply:
@lefou I added more tests.